.github/workflows/lint-test.yaml

```diff
@@ -164,20 +164,7 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0 # Fetch all git history for accurate target determination
 
-      - name: Check if dev_setup.sh has changed
```

Comment:
Need to execute `dev_setup.sh` before running the unit test so that the newest version of Boogie can be installed.